DHFR.py

```diff
@@ -0,0 +1,71 @@
+from Bio.Seq import Seq
+from Bio.SeqRecord import SeqRecord
+from Bio import SeqIO
+from Bio.SeqUtils import GC
+import numpy as np
+
+class BiologicalSequence:
```

Comment:
Поскольку BiologicalSequence должен был быть абстрактным классов, его нужно наследовать от ABC
```suggestion
class BiologicalSequence(ABC):
```